terraform/config/dev.tfvars

```diff
@@ -0,0 +1,37 @@
+environment = "dev"
+aws_region = "us-east-1"
+account_number = "911061262852"
+vpc_id = "vpc-40a23b3b"
+subnet_ids = ["subnet-1d181f40", "subnet-b29f87d6"]
+# tianna can you check these tags?
+data_providers = [{
```

Comment:
Tianna do these look ok?

Reply:
Project tag values should be in all caps (CVP, FMI)

Reply:
If that's what you're using these for...